guides/V1_ConfigurationAction.md

```diff
@@ -0,0 +1,78 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'configuring-app'
+is: 'experimental'
+---
+
+## Configuring an app
+
+There are two possible configuration actions for your app, `space-app-config-requested` and `team-app-config-requested`, 
+both are **Explicit Actions**. 
+You can learn more about the concepts and basics of action handlers, by reading [Actions & Events](./V1_ActionsEvents.md).
+
+
+## Configuring an app for a space
+
+Having a space configuration can benefit your App in many ways. For example, a scheduling app might need calendar access. 
+Having the user's authorization or credentials is essential for coordinating with third-party services.  
+
+Recognizing user-specific preferences can also enhance your App.  For example, the scheduling app can propose better meeting 
+times when user has already entered their preferred times.
+
+You can even consider customizing your app differently for each space it's been added to.   
+
+A user configuration can take place at two times:
+ - when your App is added to a space
+ - when a member of a space configures your App in that space
+
+If your App registers a Client Action Handler URL for configuring a space, then users adding it to a space will see a Configure button if they have the `space.app.configure` permission.
```

Comment:
Not sure of the need to state this permission at this point as it is not described anywhere and there is nothing developers can do with this particular permission